app/src/main/java/atorch/statspuzzles/PuzzleSelection.java

```diff
@@ -12,7 +12,7 @@
 
 public class PuzzleSelection extends AppCompatActivity {
 
-    public static final String LEVEL = "atorch.statspuzzles.LEVEL";
+    public static final String LEVELLLLL = "atorch.statspuzzles.LEVEL";
```

Comment:
This also caused an github action error, as desired:

![Screenshot from 2022-07-09 09-12-27](https://user-images.githubusercontent.com/1395971/178113851-d2753598-5861-4e81-a64d-63beefa48824.png)